solutions/devsprint-1/DeliveryAppChallenge/Screens/Home/HomeView.swift

```diff
@@ -78,6 +78,9 @@ extension HomeView: UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: "RestaurantCellIdentifier", for: indexPath) as! RestaurantCell
+        let restaurant = viewModel.restaurants[indexPath.row]
+            cell.setup(with: .init(name: restaurant.name, type: restaurant.category, deliveryTimeMin: restaurant.deliveryTime.min, deliveryTimeMax: restaurant.deliveryTime.max))
```

Comment:
Dúvida, porque passamos uma propriedade por vez ao invés de passar toda a model?

Reply:
Não sei se entendi muito bem sua dúvida. Mas sua dúvida me parece ser o por que não passamos um objeto único, talvez um `RestaurantListResponse` para a configuração da cell.
Aqui nesse caso a nossa view model da nossa HomeView não deveria ser um objeto de response como estamos fazendo aqui, a view não deve conhecer camadas de domínio e também de data (que é nesse caso), porém deixamos dessa forma mais pela questão de tempo mesmo. 
Mas de qualquer forma o ideal poderia ser uma mesma view model com tipos primitivos usado tanto pela nossa HomeView, que vai ser passada para nossa cell.

Se não for essa a dúvida, pode seguir aqui na thread!